src/components/App/index.js

```diff
@@ -45,6 +49,9 @@ class App extends Component {
   }
 
   renderBody() {
+    if (!this.props.requests) {
+      this.props.fetchPrs(this.props.oauthToken, this.props.loggedInUser);
```

Comment:
The proper place would certainly be componentWillMount, and I will move it there, but long-term this will probably end up being delegated to a child component.  The goal of this PR was just to get basic github integrations working, and this was a good way to verify that.